File: FavoriteFood.py
food_preferences = {
    "John": "Pizza",
    "Bob": "Burger",
    "Alice": "Pasta"
}

def get_favorite_food(name):
    """Returns the favorite food of the given name or 'Unknown' if not found."""
    return food_preferences.get(name, "Unknown")

def main():
    name = input("Enter the name: ").strip()
    favorite = get_favorite_food(name)
    print(f"{name}'s favorite food is: {favorite}")

if __name__ == "__main__":
    main()

File: FavoriteFood.java
import java.util.HashMap;
import java.util.Map;
import java.util.Scanner;

public class FavoriteFood {
    private Map<String, String> foodPreferences;

    public FavoriteFood() {
        foodPreferences = new HashMap<>();
        foodPreferences.put("John", "Pizza");
        foodPreferences.put("Bob", "Burger");
        foodPreferences.put("Alice", "Pasta");
    }

    public String getFavoriteFood(String name) {
        return foodPreferences.getOrDefault(name, "Unknown");
    }

    public static void main(String[] args) {
        FavoriteFood favoriteFood = new FavoriteFood();
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the name: ");
        String name = scanner.nextLine();

        System.out.println(name + "'s favorite food is: " + favoriteFood.getFavoriteFood(name));
    }
}
